### ISSUE TITLE
Query with alias fails to export, table does not exist.

### ISSUE THREAD

**Issue by tomsommer** [phpmyadmin/phpmyadmin#13222]
Related to #11509 - phpMyAdmin 4.7.0

This query fails to export

```sql
SELECT invoice_number, product_quantity, product_name
FROM  orders ord
    , order_detail det
    , product pro
WHERE invoice_number BETWEEN X AND X
and ord.id_order = det.id_order
and det.product_id = pro.id_product
ORDER BY product_name, invoice_number
LIMIT 0 , 100
```

It complains that the table `ordersord` doesn't exist.

Why is a download even started when an export query fails? It should fail with an error, not with a download containing the errorpage.

**Comment by akashkumardce** [comment]
https://github.com/phpmyadmin/phpmyadmin/pull/13296 - Fixes the Issue, actually it is due to removing all the alias while exporting, which causes issue when a table is used twice in query.

**Comment by conet** [comment]
This is still not working in latest stable 4.7.8 if the table name must be always escaped, steps to reproduce:
1. ```CREATE TABLE `temp`.`test-table` ( `a` INT NOT NULL , `b` INT NOT NULL ) ENGINE = InnoDB;
INSERT INTO `test-table` (`a`, `b`) VALUES ('1', '2')```
2. ```SELECT * FROM `test-table` t order by t.a desc ```
3. Choose export (any format)
4. ```Error reading data for table temp.test-table: #1054 - Unknown column 'test' in 'order clause'``` and the badly generated query is ```SELECT * FROM `test-table` order by test-table.a```

**Comment by whiteatom** [comment]
I'm having the same issue.. but it doesn't let me export. When I click on the "Export" link, I'm taken to the select page. 

I have noticed, that it's only a problem when the table at the top of the page shows my aliased table name.

**Comment by conet** [comment]
Still present in *4.8.2*.

**Comment by williamdes** [comment]
Duplicate of #12241

**Comment by williamdes** [comment]
Will be fixed as #12241
